rails-commenting.rb

```diff
@@ -5,27 +5,27 @@
 
 # app/controller/blog_posts_controller.rb
 
-# 1)
+# 1)This indicates the controller that we are in, in this case it controls the blog posts for the app
```

Comment:
Great! I'd additionally elaborate on how it is inheriting the ApplicationController